membership-attribute-service/conf/CODE.public.conf

```diff
@@ -0,0 +1,3 @@
+include "DEV.public"
+
+logstash.enabled=true
```

Comment:
I've updated the CODE props in S3 to point to this file instead of the DEV one as they did before